### PR TITLE
Create fake rpms for golang

### PR DESCRIPTION
**Pull request by smarterclayton**

### CODE REVIEW

**Comment by smarterclayton** [comment]
@stevekuznetsov [test]

File: images/release/fake-rpm.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+tmpdir=$(mktemp -d -t rpmspecXXXXXX)
+HOME=$tmpdir rpmbuild -bb --define "_topdir $tmpdir/rpmbuild" golang.spec
```
Comment (stevekuznetsov) [inline images/release/fake-rpm.sh]:
```sh
HOME="${tmpdir}" rpmbuild -bb --define "_topdir ${tmpdir}/rpmbuild" golang.spec
     ^ ^      ^^                                 ^      ^
```

**Comment by stevekuznetsov** [comment]
@sdodson you should probably take a look

**Comment by sdodson** [comment]
Neat, seems like an alright situation to your problem. I can't think of anything better.

**Comment by smarterclayton** [comment]
I tried to create fake rpms containing the golang zip and ran into a ton of problems, so probably just not do that in the future.

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**